src/components/Explore/TokenRow.tsx

```diff
@@ -17,20 +18,26 @@ import {
   MOBILE_MEDIA_BREAKPOINT,
   SMALL_MEDIA_BREAKPOINT,
 } from './constants'
-import { favoritesAtom, useToggleFavorite } from './state'
+import {
+  favoritesAtom,
+  filterTimeAtom,
+  sortCategoryAtom,
+  sortDirectionAtom,
+  useSortCategory,
+  useToggleFavorite,
+} from './state'
 import { TIME_DISPLAYS } from './TimeSelector'
 
-enum Category {
+export enum Category {
```

Comment:
lets make a new file structure:
```

- Explore
    - TokenTable
        - index.tsx
        - types.tsx
        - Row.tsx
    -  TokenDetails
        - index.tsx
        - ...
    - state.ts
    - types.ts
```

Then move these shared types into the right places. If it's Table specific put it in that subfolder, else you can put in the top level `types.ts`.